Outlines/simple-tcr.md

```diff
@@ -0,0 +1,18 @@
+# Proposed title
+Simple Token-Curated Registry
```

Comment:
@Destiner Titles should be more descriptive and explain what the reader is going to gain from reading this, so how about 

```suggestion
Creating incentivized lists with a Token-Curated Registry
```

or something like this…

Reply:
What do you think of
1) Incentivizing high-quality curation with a Token-Curated Registry
2) Incentivizing item selection with a Token-Curated Registry?

Reply:
@Destiner Number 1.

Reply:
Changed the title.